packages/m/mfast/patches/v1.2.2/tinyxml2.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -62,7 +62,11 @@ endif(BUILD_PACKAGES)
+ 
+ find_package(Boost 1.56.0 REQUIRED)
+ include_directories(${Boost_INCLUDE_DIR})
+-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/tinyxml2)
++
++find_package(tinyxml2 REQUIRED)
++include_directories("${tinyxml2_DIR}/../../../include/")
++link_directories("${tinyxml2_DIR}/../../")
```

Comment:
TinyXML2 do not have any proper CMake variables to include/link:

`> -- tinyxml2_CONFIG=/usr/local/lib/cmake/tinyxml2/tinyxml2-config.cmake
> -- tinyxml2_CONSIDERED_CONFIGS=/usr/local/lib/cmake/tinyxml2/tinyxml2-config.cmake
> -- tinyxml2_CONSIDERED_VERSIONS=9.0.0
> -- tinyxml2_DIR=/usr/local/lib/cmake/tinyxml2
> -- tinyxml2_FOUND=1
> -- tinyxml2_VERSION=9.0.0
> -- tinyxml2_VERSION_COUNT=3
> -- tinyxml2_VERSION_MAJOR=9
> -- tinyxml2_VERSION_MINOR=0
> -- tinyxml2_VERSION_PATCH=0
> -- tinyxml2_VERSION_TWEAK=0
> -- tinyxml2_comp=
> -- tinyxml2_comp_shared=NO
> -- tinyxml2_comp_static=NO
> -- tinyxml2_known_comps=static;shared
> -- tinyxml2_shared_targets=/usr/local/lib/cmake/tinyxml2/tinyxml2-shared-targets.cmake
> -- tinyxml2_static_targets=/usr/local/lib/cmake/tinyxml2/tinyxml2-static-targets.cmake`